docs/developer-guide/cicd/helpers/README.md

```diff
@@ -47,17 +38,41 @@ namespace: io.kestra.tests
 tasks:
 - id: t1
   type: io.kestra.core.tasks.debugs.Return
-  format: "[[> lorem.txt]]"
+  format: |
+    [[> lorem.txt]]
 ``` 
 And have a local file `lorem.txt` with the large content in it. 
 
-The path can be: 
-* `[[> lorem.txt]]`: a relative path from the flow (flow.yaml and lorem.txt are on the same directory),
+The path can be:
+* `[[> lorem.txt]]`: a relative path from the flow (flow.yaml and lorem.txt are in the same directory),
 * `[[> /path/to/lorem.txt]]`: an absolute path,
 * `[[> path/to/lorem.txt]]`: a relative path from the flow (flow.yaml with a subdirectory `path/to/`).
 
+When including a file, you must use the right YAML scalar type: literal (with or without quotes) for single-line scalars or folded for multiple-lines ones.
+
 
 ::: warning
 Includes are resolved recursively, so you can include a file from another include. 
-Since this allows more complex things, you need to take care that included files don't contain `[[ .. ]]` . If you need to have these characters in included files, escape them with `\[[ ...]]` !
+This allows more complex things, but you need to take care that included files don't contain `[[ .. ]]`. If you need to have these characters in included files, escape them with `\[[ ...]]` !
 :::
+
+
+## Validate the flow to be uploaded to the server
+
+There is a convenient command on the Kestra executable that allows validation of the current flow:
+
+```bash
+./kestra flow validate --local path-to-your-flow.yaml 
+```
+::: tip
+If your flow uses a helper function, flow validation must be done locally as the flow cannot be expanded on the webserver.
```

Comment:
Precise that local kestra must have plugins used in the flow to validate it.

Reply:
Done